frontend/src/components/CircularTimer.vue

```diff
@@ -33,7 +33,7 @@
         return {
           secondsPlusOne: this.seconds + 1,
           fontSize: ((this.size - (this.lineThickness * 2)) / 2) + 'px',
-          halfSize: (this.size / 2) + 'px',
+          halfSize: (this.size / 2).toString() + 'px',
```

Comment:
The idiomatic javascript way to do this would be

```js
`${this.size / 2}px`
```

Reply:
Agreed